src/main/java/com/microfocus/application/automation/tools/sv/Utils.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Certain versions of software accessible here may contain branding from Hewlett-Packard Company (now HP Inc.) and Hewlett Packard Enterprise Company.
+ * This software was acquired by Micro Focus on September 1, 2017, and is now offered by OpenText.
+ * Any reference to the HP and Hewlett Packard Enterprise/HPE marks is historical in nature, and the HP and Hewlett Packard Enterprise/HPE marks are the property of their respective owners.
+ * __________________________________________________________________
+ * MIT License
+ *
+ * Copyright 2012-2024 Open Text
+ *
+ * The only warranties for products and services of Open Text and
+ * its affiliates and licensors ("Open Text") are as may be set forth
+ * in the express warranty statements accompanying such products and services.
+ * Nothing herein should be construed as constituting an additional warranty.
+ * Open Text shall not be liable for technical or editorial errors or
+ * omissions contained herein. The information contained herein is subject
+ * to change without notice.
+ *
+ * Except as specifically indicated otherwise, this document contains
+ * confidential information and a valid license is required for possession,
+ * use or copying. If this work is provided to the U.S. Government,
+ * consistent with FAR 12.211 and 12.212, Commercial Computer Software,
+ * Computer Software Documentation, and Technical Data for Commercial Items are
+ * licensed to the U.S. Government under vendor's standard commercial license.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ___________________________________________________________________
+ */
+
+package com.microfocus.application.automation.tools.sv;
+
+import hudson.model.*;
+import hudson.security.ACL;
+import hudson.security.AuthorizationStrategy;
+import hudson.security.FullControlOnceLoggedInAuthorizationStrategy;
+import jenkins.model.Jenkins;
+
+public final class Utils {
+    private Utils() {
+        // no meaning instantiating
+	}
+
+    public static boolean hasCurrentUserConfigurePermission() {
+        Jenkins jenkins = Jenkins.getInstanceOrNull();
+        if (jenkins == null) {
+            return false;
+        }
+        AuthorizationStrategy authStrategy = jenkins.getAuthorizationStrategy();
+        if (authStrategy == null) {
+            return false;
+        }
+        User user = User.current();
+        if (authStrategy instanceof FullControlOnceLoggedInAuthorizationStrategy) {
+            return user != null;
+        }
+        if (user == null) {
+            user = User.getUnknown();
+        }
+        ACL acl = authStrategy.getACL(user);
+        return acl.hasAnyPermission(Jenkins.ADMINISTER, Item.CONFIGURE);
```

Comment:
Permissions like `Item.CONFIGURE` must be checked against the specific item, not globally. See e.g. https://www.jenkins.io/doc/developer/security/form-validation/#checking-permissions for documentation.

Reply:
Hi @daniel-beck, 
Thank you so much for your suggestion...
We'd appreciate if you can advise a little bit about this:
We are searching a way to pass the Item object to a standard getter function like this:
https://github.com/jenkinsci/hpe-application-automation-tools-plugin/pull/660/files#diff-9d4c19bacd81915ab09939bb6ac6266105e95c4de1f24c27b0f4bf2ec3dfcb94R71

called manually from config.jelly, like this:
https://github.com/jenkinsci/hpe-application-automation-tools-plugin/pull/660/files#diff-8cf610d3ccce94df96e12fbb4e592504c890bc4abe8054e6394d4fb4c3016383R39

Is it possible?

Reply:
Hi @daniel-beck , We have changed the checking mechanism for the permission and we would appreciate your input.